### DISCUSSION TITLE
I can't use Swiper.js when i install through NPM

### DISCUSSION THREAD

**Discussion by ViniciusMoreira33** [nolimits4web/swiper#5028] [Q&A]
For some reason, i can't use swiper when i install using npm, only works using CDN. im really getting upset because i think i'm missing something very obvious but just cant figure out what is exactly. To test it, i created a new project with the essentials: index.html, style.css and app.js. Used npm init and then installed swiper and finnaly copy-pasted exactly the code of the documetation.

My guess is that im forgetting or missing something about npm. Is that it or is it something else?

Any help is apreciatte :)

**Comment by awacode21** [comment]
Can you share your reproduction?
